docs/developer-center/guides/06-Explore-the-Data-Observatory-catalog.md

```diff
@@ -0,0 +1,5614 @@
+      <td>None</td>
+    </tr>
+    <tr>
+      <th>51</th>
+      <td>SUM</td>
+      <td>AGECY3539</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>INTEGER</td>
+      <td>Population age 35-39 (2019A)</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>AGECY3539</td>
+      <td>AGECY3539_fed2aa71</td>
+      <td>False</td>
+      <td>{'head': [0, 1, 0, 1, 0, 0, 0, 0, 0, 0], 'tail...</td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <th>52</th>
+      <td>SUM</td>
+      <td>EDUCYASSOC</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>INTEGER</td>
+      <td>Pop 25+ Associate degree (2019A)</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>EDUCYASSOC</td>
+      <td>EDUCYASSOC_fa1bcf13</td>
+      <td>False</td>
+      <td>{'head': [0, 0, 0, 1, 3, 0, 0, 0, 0, 0], 'tail...</td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <th>55</th>
+      <td>SUM</td>
+      <td>POPPY</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>FLOAT</td>
+      <td>Population (2024A)</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>POPPY</td>
+      <td>POPPY_946f4ed6</td>
+      <td>False</td>
+      <td>{'head': [0, 0, 8, 0, 0, 0, 4, 0, 2, 59], 'tai...</td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <th>66</th>
+      <td>SUM</td>
+      <td>SEXCYMAL</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>INTEGER</td>
+      <td>Population male (2019A)</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>SEXCYMAL</td>
+      <td>SEXCYMAL_ca14d4b8</td>
+      <td>False</td>
+      <td>{'head': [1, 2, 0, 13, 374, 0, 0, 0, 0, 0], 't...</td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <th>67</th>
+      <td>SUM</td>
+      <td>SEXCYFEM</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>INTEGER</td>
+      <td>Population female (2019A)</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>SEXCYFEM</td>
+      <td>SEXCYFEM_d52acecb</td>
+      <td>False</td>
+      <td>{'head': [5, 3, 0, 9, 585, 0, 0, 0, 0, 0], 'ta...</td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <th>75</th>
+      <td>SUM</td>
+      <td>POPCYGRPI</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>INTEGER</td>
+      <td>Institutional Group Quarters Population (2019A)</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>POPCYGRPI</td>
+      <td>POPCYGRPI_147af7a9</td>
+      <td>False</td>
+      <td>{'head': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0], 'tail...</td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <th>76</th>
+      <td>SUM</td>
+      <td>POPCYGRP</td>
+      <td>carto-do.ags.demographics_sociodemographic_usa...</td>
+      <td>INTEGER</td>
+      <td>Population in Group Quarters (2019A)</td>
+      <td>carto-do.
```

Comment:
I'm adding a new utility class named `u-vertical-scroll`. Please add it in the table, next to the "dataframe" class (I'll add this information in the guidelines asap)